core/app/models/spree/image/active_storage_attachment.rb

```diff
@@ -15,7 +19,15 @@ module Spree::Image::ActiveStorageAttachment
                    large: '1200x1200>'
                  },
                  default_style: :product
-    validates :attachment, presence: true
-    validate :attachment_is_an_image
+
+    def supported_content_type
+      unless attachment.content_type.in?(accepted_image_types)
+        errors.add(:attachment, :content_type_not_supported)
+      end
+    end
+
+    def accepted_image_types
+      %w(image/jpeg image/jpg image/png image/gif)
```

Comment:
What do you think about having the list of accepted mime types in a constant?

Reply:
Defined in a ENV constant or where would the constant be defined?

Reply:
No, no... I mean as a constant in `Spree::Image::ActiveStorageAttachment`, like in:

```ruby
module Spree::Image::ActiveStorageAttachment
  ACCEPTED_IMAGE_TYPES = %w(....).freeze
end
```

I mean, it's information that doesn't depend on the state of an object, and it has no computational logic, so in my view, it belongs to the constants domain.

Sorry not having been clearer.

Reply:
I see, thank you for clarifying